ansible/vars/dut_droplet_vlans-1.yml

```diff
@@ -0,0 +1,11 @@
+Vlans:
+  Vlan100: 
+   id: 100
+   intfs: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12]
+   tag: 100
```

Comment:
tag and id are synonyms. Probably it is better to make one field only?

Reply:
The minigraph has these two fields and so I simply mimicked what it provides.